test/traffic_matcher.cc

```diff
@@ -15,6 +15,12 @@
 
 using namespace valhalla;
 
+// NOTE: this test relies on pregenerated data. If you change the data format
```

Comment:
i didnt end up needing to write a test to test loading gzipped tiles because one already exists (i had an inkling that i had done that before to get a 2 for 1 test but i couldnt remember for sure  until i got a test failure haha)